packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -254,9 +254,9 @@ export class TerminalFrontendContribution implements FrontendApplicationContribu
         });
         this.mergePreferencesPromise = this.mergePreferencesPromise.finally(() => this.mergePreferences());
 
+        // extension contributions get read after this point: need to set the default profile if necessary
         this.profileService.onAdded(id => {
-            // extension contributions get read after this point: need to set the default profile if necessary
-            let defaultProfileId;
+            let defaultProfileId = undefined;
```

Comment:
Minor: This gets implicitly typed as `any`
```suggestion
            let defaultProfileId: string | undefined;
```

Reply:
Strange: it seems trivial to infer the type here.

Reply:
TypeScript doesn't seem to do it any ¯\_(ツ)_/¯